solidus_frontend.gemspec

```diff
@@ -15,6 +22,9 @@ Gem::Specification.new do |s|
   s.license     = 'BSD-3-Clause'
 
   s.metadata['rubygems_mfa_required'] = 'true'
+  s.metadata['homepage_uri'] = s.homepage
+  s.metadata['source_code_uri'] = 'https://github.com/gsmendoza/solidus_frontend'
```

Comment:
We'll need to update those URIs to point to the Solidus organziation

Reply:
Okay, I guess it doesn't hurt to set those URIs in advance :) TODO